visualization/app/codeCharta/ui/searchBar/searchBar.component.html

```diff
@@ -1,16 +1,16 @@
 <i class="fa fa-search search-icon"></i>
 
 <md-input-container md-no-float="">
-	<input
-		name="search"
-		id="searchInput"
-		type="text"
-		placeholder="Search: *.js, **/app/*"
-		ng-model="$ctrl._viewModel.searchPattern"
-		ng-change="$ctrl.handleSearchPatternChange()"
-	/>
+	<input name="search" id="searchInput" type="text" placeholder="Search: *.js, **/app/*"
+		ng-model="$ctrl._viewModel.searchPattern" ng-change="$ctrl.handleSearchPatternChange()" />
+		<div class="small-action-button transparent clear-icon">
```

Comment:
I would prefer an inline element like a span instead of the div (block element)

Reply:
Do you mean a span around the button?